infrastructure/lambda-nems-message.tf

```diff
@@ -0,0 +1,71 @@
+module "nems-message-lambda" {
+  count          = local.is_mesh_forwarder_enable ? 1 : 0
+  source         = "./modules/lambda"
+  name           = "NemsMessageLamba"
+  handler        = "handlers.nems_message_handler.lambda_handler"
+  lambda_timeout = 60
+  iam_role_policies = [
+    "arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole",
+    "arn:aws:iam::aws:policy/CloudWatchLambdaInsightsExecutionRolePolicy",
+    module.lloyd_george_reference_dynamodb_table.dynamodb_policy,
+  ]
+  rest_api_id       = aws_api_gateway_rest_api.ndr_doc_store_api.id
+  api_execution_arn = aws_api_gateway_rest_api.ndr_doc_store_api.execution_arn
+  lambda_environment_variables = {
+    WORKSPACE                = terraform.workspace
+    LLOYD_GEORGE_BUCKET_NAME = "${terraform.workspace}-${var.lloyd_george_bucket_name}"
```

Comment:
For this env var, maybe we want the Lloyd George dynamodb table name instead of the S3 bucket name? From ticket description seems that we are going to edit the record in dynamodb